src/pages/Home.jsx

```diff
@@ -0,0 +1,19 @@
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import Hero from '../components/Hero';
+import ProvinceList from '../components/ProvinceList';
+import { setHeader } from '../redux/slice';
+
+export default function Home() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(setHeader('Afghanistan'));
+  }, [dispatch]);
+
+  return (
+    <>
+      <Hero />
+      <ProvinceList />
+    </>
+  );
+}
```

Comment:
- [ ] On the home page you failed to add a search/filter feature. Kindly add this. Users should be filter elements on the home page.

  ![image](https://github.com/M-Anwar-Hussaini/Weather-Forecast/assets/54848363/8a90e8d6-19da-44ac-a1e2-043b626ee3da)